src/sources/kubernetes/mod.rs

```diff
@@ -283,6 +163,7 @@ fn transform_pod_uid() -> crate::Result<ApplicableTransform> {
 
     let namespace_regex = r"(?P<pod_namespace>[0-9a-z.\-]*)";
     let name_regex = r"(?P<pod_name>[0-9a-z.\-]*)";
+    // TODO: rename to pod_uid?
```

Comment:
can we resolve this todo in this PR? What is stopping us?

Reply:
I forgot about that todo, and I meant it as a question. The reason being that the first `pod_uid` that we encounter is defined by CRI, but this one is UID of a pod as defined by Kubernetes API. While naming it as `object_uid` is correct, we could be more specific. But I don't like the idea of having two fields of different meaning with the same name.

This question also touches on the Kubernetes metadata transform, and yet unresolved question that you posed, regarding should we put metadata fields under `kubernetes` key. So I think we should deal with this renaming in that PR.

Reply:
sounds good to me :+1: